src/core/config/config.service.ts

```diff
@@ -91,6 +91,13 @@ export class ConfigService {
     };
   }
 
+  @Lazy() get passwordSecret() {
+    return Buffer.from(
```

Comment:
Would you mind doing this conversion has a private getter/method in the AuthenticationService? I don't think the fact that argon2 needs it as a buffer should concern the config service.